src/components/portal/portal.tsx

```diff
@@ -67,26 +57,23 @@ export class Portal {
     public position: 'fixed' | 'absolute' = 'absolute';
 
     /**
-     * A unique ID.
+     * @deprecated no longer needed or used, will be removed in a future release
      */
     @Prop({ reflect: true })
     public containerId: string;
 
     /**
      * Dynamic styling that can be applied to the container holding the content.
+     *
+     * @deprecated still used in the component for backwards compatibility, but
+     * is no longer needed since the consumer can just style the slotted
+     * component as usual
      */
     @Prop()
     public containerStyle: object = {};
 
     /**
-     * The `parent` property specifies the parent element where the content
-     * of the portal will be moved to.
-     * By default, it is set to `document.body`, meaning the content
-     * will be appended as a child of the body element in the DOM.
-     * If you want the content to be appended to a different element,
-     * you can specify that element by setting this property.
-     * Please note that the specified parent element should exist
-     * in the DOM at the time of rendering the portal.
+     * @deprecated no longer needed or used, will be removed in a future release
```

Comment:
it's only us using limel-portal inside lime elements. Nobody knows about this `@private` component. You don't need to `deprecate` it. ;)

Reply:
We can deprecate it anyway :smile: